### ISSUE TITLE
Use Mammal Diversity Database of the American Society of Mammalogists (ASM)

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#408]
**Dataset title**
The Mammal Diversity Database of the [American Society of Mammalogists](http://www.mammalsociety.org/) (ASM)

**Dataset contact & access**
https://www.mammaldiversity.org/index.html
There exists a basic CSV download, but that misses synonyms, vernacular names and distribution data which is available on their site.

**Taxonomic group & CoL sector**
Mammalia

**Motivation for changing/adding**
A continuously maintained dataset for all mammals of the worl incl. extinct ones since ~1500 CE.
The dataset contains 6,591 species as of today while COL via ITIS contains just 6025

**Comment by mdoering** [comment]
https://github.com/CatalogueOfLife/data/issues/570

**Comment by DaveNicolson** [comment]
Have the issues with MDD synonymy been resolved yet? E.g., lack of explicit rank and combinations for ALL synonyms (they're just given as an unranked epithet & authorship with parens as if it had been published in combination with the current genus, rather than simply having been synonymized)? Until that issue is resolved, switching from ITIS will get you much quicker current data, but you'll sacrifice a lot on synonymy. That's a trade-off, so the question is whether synonymy based on names published in the literature is "more valuable" than always having more current valid names, I guess, FWIW.

I feel like a broken record. Every time a mammal record isn't current there's a cry for moving to MDD, but nobody seems to be paying attention to the synonymy issue with MDD.

**Comment by DaveNicolson** [comment]
Additionally, it should be noted that MDD lacks support for valid subspecies. So just species counts is not the whole story... For valid/accepted mammal species & subspecies, ITIS contains:

usage|rank_name|validCount
valid|Species|6037
valid|Subspecies|5980 [of which 1237 are nominotypical, if you want to adjust species or subspecies counts accordingly]

So yes, you'd get the current (April 2023 version of MDD) 6649 valid/accepted species, but you'd lose almost all synonymy, almost all vernaculars, and all the thousands of subspecies. I do hope MDD can overcome these issues, but as of now, there are significant pros and cons to weigh.

**Comment by mdoering** [comment]
Missing subspecies and synonyms is indeed a show stopper.
But when I look at a taxon page on their site the "Nominal names" looks a lot like a synonymy:
http://www.mammaldiversity.org/taxon/1006393

Maybe @n8upham knows how complete synonymy and subspecific information in MDD is?

**Comment by n8upham** [comment]
Hi @mdoering, thanks for tagging me in here -- yes, the MDD's nominalNames field is increasingly contains synonymy information for the accepted species in question.  This is thanks to some stellar curation work that's been done by @connorjburgin and @JelleZijlstra over the past ~2 years.  

Starting with the v2.0 release of the MDD taxonomy, we will be publishing a curated list of over 50k species-level synonyms, including citation, type locality, and type specimen information for most names.  This release is being accompanied by a new article describing the updates since v1.0 (2018) that we will soon be submitting to Journal of Mammalogy -- so the likely v2.0 release will be Feb 2025.  Once that happens, moving COL infrastructure toward the MDD as a reference source will become much more feasible.

@DaveNicolson -- the MDD is not planning to yet include accepted subspecies, but those subspecies names are included under the species-level synonyms so nonetheless traceable.  Would that alleviate ITIS' concerns as well?

Thanks, --nate

**Comment by JelleZijlstra** [comment]
Note that my Hesperomys database (https://hesperomys.org; regular data exports at https://zenodo.org/records/13824506) uses the same classification and synonymy as the MDD, but also includes subspecies and already publishes full synonyms for all species.

I am not aware of the exact constraints here but I'd be happy to help integrate the MDD and/or Hesperomys data into Catalogue of Life.

**Comment by JelleZijlstra** [comment]
Yes, on the website I attribute name combinations to the first author to use that name combination; that could perhaps be presented better. If there is a specific guideline on how the data should be laid out, I'd be happy to generate a data export in the format you need.

**Comment by DaveNicolson** [comment]
Actually, if you're going to present the "usage" authorship, as distinct from the expected protonym-based authorship, some kind of special punctuation (like a colon or various types of dashes or even a comma) should be placed between the name and the author who first used it (but who didn't describe/name the nominal taxon in question), as suggested by [Gardner & Hayssen (2004)](https://repository.si.edu/handle/10088/3785), which specifically says to use a colon between the final epithet and the "usage" author in that sort of synonymy. Otherwise I would suggest you follow the ICZN's Recommendation 51G, and the rest of [Article 51](https://code.iczn.org/authorship/article-51-citation-of-names-of-authors/), so that the nominal taxon author is provided for each name & combination (as in my example above starting with Mus bimaculatus) and the "usage" authorship/citation can be provided after, if you like.

In summary, the "core" of name and its Code-based authorship (with parentheses or not, per Art. 51.3) is needed for all names/combinations in synonymy; additional info like publication citation that it appeared in, is additional valuable information that's easiest to digest in COL (I think?) as a reference/publication citation linked to the name.

I am not sure how much of this should apply to your website format and how much to an export format for COL and/or ITIS.

That's my take / 2 cents....

**Comment by mdoering** [comment]
I would strongly ask to prefix the usage author with `sec` or `sensu` which is the standard way to do so. And which is supported by our and many other software in contrast to colons which I come across here for the first time. Thanks @DaveNicolson for the above link to Gardner

**Comment by JelleZijlstra** [comment]
Thanks for that feedback; I'll make changes here the next time I update the website.

**Comment by DaveNicolson** [comment]
I guess it might depend on whether you're trying to track taxonomic "concepts", or whether you're simply trying to cite the source that first used the name/combination... In one case the sensu/sec approach seems to make good sense, while in the other a simple citation/link between the name and the work that used it could be more useful. I am not strongly versed in the sensu/sec approach, so if this seems wrong feel free to correct me.

**Comment by mdoering** [comment]
Version 2 of MDD has come out which includes many species synonyms:
https://www.mammaldiversity.org/

>A comprehensive nomenclatural dataset for 50,000+ synonymous species-rank names, curated with type locality and specimen information.

**Comment by n8upham** [comment]
Amen, glad you found that @mdoering !  Yes, the MDD v2.0 species synonym file should be useful for you all [Species_Syn_v2.0.csv](https://zenodo.org/records/15007505/files/Species_Syn_v2.0.csv?download=1)) -- but any feedback that you have about the format of it would be welcomed.  I don't think we are DwC compliant yet, but would like to be for maximal interoperability.  Could you provide some guidance in that direction?  Cheers, --n

**Comment by mdoering** [comment]
Ideally you would already provide a full dwc archive or [ColDP](https://github.com/CatalogueOfLife/coldp/blob/master/README.md) file.

Otherwise the columns seem rather clear to me and straight forward to do the transformation.
I will send you a ColDP file once I have that done, probably next week.

**Comment by dhobern** [comment]
@mdoering Is there a COLDP dataset yet - should the TG be carrying out a review?

**Comment by mdoering** [comment]
I start working on one tomorrow. Currently we still only have the old version in CLB

**Comment by n8upham** [comment]
@mdoering We just today launched the MDD v2.1 taxonomy to make up for the long lag on the v2.0 taxonomic cutoff (from 15 Aug 2024, to allow us time to write that manuscript).  The v2.1 taxonomy is downloadable here (https://www.mammaldiversity.org/) and on the Zenodo archive (https://doi.org/10.5281/zenodo.4139722).  Let me know if you want me to review the CoL conversion once you're done.  Thanks!!

**Comment by mdoering** [comment]
I have converted most of the 2.1 version into a ColDP file and uploaded it into CLB:
https://www.checklistbank.org/dataset/9802/imports

In the synonyms I had encountered quite a few oddly capitalised names, e.g. `Antilope Mazama` or `otaria coronata`, which I all converted into the standard Capitalised form.

Distributions are missing at this point, I'll try to add them later today or tomorrow. They are by country, biogeographic realm & with notes.

**Comment by mdoering** [comment]
@n8upham do the synonym names also contain verbatim spellings with upper case epithets and lower case genera or are those problems I should report?

For example "Ursus Arctos" here: https://www.mammaldiversity.org/taxon/1005935/

**Comment by JelleZijlstra** [comment]
It's intentional that the synonyms contain non-normalized case formations, yes. You may encounter other non-normalized synonyms, e.g. using ligatures like æ, hyphens in the species name, or even a space in the specific name.

In my own database that the MDD synonym sheet is based on (Hesperomys), I use two different columns for the normalized and non-normalized original combination. We could consider making the MDD use the normalized original combination, or present both.

**Comment by n8upham** [comment]
Yeah nice, thanks Jelle. It's a good point that Marcus raises that most
folks expect to see the normalized synonyms even if those weren't how the
usages were originally presented. Adding both normalized and non-normalized
columns to the MDD to mirror Hesperomys more closely makes sense. And
likely what CoL / GBIF are expecting to see.

**Comment by mdoering** [comment]
I am normalising the form myself when converting to ColDP and that seems to work fine. Name parsers get rather confused about Camel Case names as a lower case epithet is one of the few things that distinguishes them from author names. [Ursus Arctos](https://api.checklistbank.org/parser/name?name=Ursus%20Arctos) is usually parsed as a genus with the author being Arctos. 

Storing the original spelling seems a good practise to me, just wanted to make sure ;)

**Comment by JelleZijlstra** [comment]
I'll try to make it so we add a separate column for the normalized original name in the MDD in the future.

In my experience it's not always possible to automatically go from the non-normalized to the normalized name; for example, some specific names were written as multiple words ("_Rhinolophus ferrum equinum_", "_Hystrix africae australis_").

**Comment by camiplata** [comment]
@mdoering can we make a test of MDD for the extended release?

**Comment by n8upham** [comment]
Sounds like we've addressed some of the previous concerns about the MDD?  @JelleZijlstra @mdoering @DaveNicolson 
To refresh, these issues were 
(1) synonyms not normalized as given -- starting with MDD v2.2 we now added "MDD_normalized_original_combination" as a column in the [Species_Syn_v2.2.csv](https://zenodo.org/records/15659662/files/Species_Syn_v2.2.csv?download=1) file. We are now up to 59,436 synonyms and name combinations.

(2) name combination authors confused with epithet authors -- I think this is solved just by using the "MDD_nomenclature_status" column with entries of "name_combination" not being epithet authors while entries of "available" are the original epithet authors.  The rest is in how you interpret / parse that info.  E.g., 

MDD_syn_ID | MDD_species | MDD_root_name | MDD_author | MDD_year | MDD_authority_parentheses | MDD_nomenclature_status | MDD_validity | MDD_original_combination
-- | -- | -- | -- | -- | -- | -- | -- | --
100022091 | Abeomelomys sevia | sevia | Tate & Archbold | 1935 | 1 | available | species | Melomys sevia
100022092 | Abeomelomys sevia | tatei | Hinton | 1943 | 1 | available | synonym | Pogonomelomys tatei
100035235 | Abeomelomys sevia | sevia | Tate | 1951 | 1 | name_combination | synonym | Pogonomelomys sevia
100044950 | Abeomelomys sevia | tatei | Tate | 1951 | 1 | name_combination | synonym | Pogonomelomys sevia tatei
100057154 | Abeomelomys sevia | sevia | Laurie & J. Edwards Hill | 1954 | 1 | name_combination | synonym | Pogonomelomys sevia sevia
100040581 | Abeomelomys sevia | sevia | Menzies | 1990 | 0 | name_combination | synonym | Abeomelomys sevia

(3) subspecies not recognized formally -- this is still not present in MDD v2.2, though we are starting to curate this information and did include a "subspecies" column in the main taxonomy release this time (still very preliminary, only filled for 1429 species so far-- [MDD_v2.2_6815species.csv](https://zenodo.org/records/15659662/files/MDD_v2.2_6815species.csv?download=1)).  Arguably, formal subspecies recognition across mammals is a fruitless (and misleading) task given their varied uses across higher taxa, but we do recognize that some groups (e.g., bats, marine mammals) use these categories more routinely.

Also any guidance you have about generating DwC releases of these files on our end would be welcomed.  Thanks!!

**Comment by mdoering** [comment]
> [@mdoering](https://github.com/mdoering) can we make a test of MDD for the extended release?

sure!
